bin/PerlDDS/Run_Test.pm

```diff
@@ -611,7 +618,20 @@ sub setup_discovery {
   my $params = shift;
   my $executable = shift;
   $params = "" if !defined($params);
-  $executable = "$ENV{DDS_ROOT}/bin/DCPSInfoRepo" if !defined($executable);
+  if (!defined($executable)) {
+    $executable = "$ENV{DDS_ROOT}/bin/DCPSInfoRepo";
+    if (!(-e $executable) && !(-e "${executable}.exe")) {
+      if (!defined($ENV{OPENDDS_INSTALL_PREFIX})) {
+        print STDERR "ERROR: Couldn't find \$DDS_ROOT/bin/DCPSInfoRepo. It " .
+          "needs to be built or \$OPENDDS_INSTALL_PREFIX needs to be defined " .
+          "if OpenDDS is installed.\n";
+        $self->{status} = -1;
+        return;
```

Comment:
I know I referenced this in my other comment, but is this going to stop tests when this error is triggered?